config/environment.rb

```diff
@@ -8,22 +8,30 @@
 require 'sinatra/activerecord'
 require 'sinatra/contrib/all' # Requires cookies, among other things
 
-require 'pry'
 
+
+ENV['RACK_ENV'] ||= 'development'
```

Comment:
Probably not needed. If it was it should be done before `require 'sinatra'` anyway.

If `$RACK_ENV` is unset, `Sinatra::Application.environment` will return `:development` anyway. So as long as you refer to that and not `ENV['RACK_ENV']` then there's no need to change the environment variable.